apps/coscrad-frontend/src/components/transcripts/transcript-items-presenter.tsx

```diff
@@ -0,0 +1,35 @@
+import { ITranscriptItem } from '@coscrad/api-interfaces';
+import { Box, Typography } from '@mui/material';
+import { MultilingualTextItemPresenter } from '../../utils/generic-components/presenters/multilingual-text-item-presenter';
+
+interface TranscriptItemsPresenterProps {
+    transcriptItems: ITranscriptItem[];
+}
+
+export const TranscriptItemsPresenter = ({
+    transcriptItems,
+}: TranscriptItemsPresenterProps): JSX.Element => {
+    if (transcriptItems.length === 0) return null;
+
+    return (
+        <>
+            {transcriptItems.map(
+                ({ speakerInitials, inPointMilliseconds, outPointMilliseconds, text }) => (
+                    <Box key={`${speakerInitials}-${inPointMilliseconds}`}>
+                        <Typography component="span" variant="body1" mr={1} fontWeight="bold">
+                            {speakerInitials} [{inPointMilliseconds}-{outPointMilliseconds}]:
+                        </Typography>
+                        {text.items.map(({ text, languageCode, role }) => (
+                            <MultilingualTextItemPresenter
```

Comment:
Could we do this?  Or should I go back to just the item as string?

Reply:
You absolutely need to expose the text and any translations to the users. I don't know that we want to use the same presenter with the accordion, though. It's also possible to define a different multilingual presenter for this purpose. In a print transcript, for each item, we would typically see something like:
```
[22:34] [AP]
This is what was said in the language.

# can be collapsed
This is the translation into English
[24:02]
```

Reply:
I've left this as a simple string for now so that we can address it in a separate PR.